### PR TITLE
Third branch - Week 3: JSON

### PR DESCRIPTION
**Pull request by kasimfareeda**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,7 +14,12 @@
 
 package com.google.sps.servlets;
 
+//import com.google.sps.data.ServerStats;
```
Comment (setareh94) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:17]: remove?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,7 +14,12 @@
...
+import com.google.gson.Gson;
```
Comment (setareh94) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:18]: Don't you need to add this to your pom.xml file 
```<dependency>
  <groupId>com.google.code.gson</groupId>
  <artifactId>gson</artifactId>
  <version>2.8.6</version>
</dependency>```

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -26,7 +31,31 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Fareeda!</h1>");
+
+    ArrayList<String> strings = new ArrayList<String>();
+    strings.add("Hello");
```
Comment (setareh94) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:36]: Did you mean to add different sentences to this list?

Similar to?
```
    quotes = new ArrayList<>();
    quotes.add(
        "A ship in port is safe, but that is not what ships are for. ");
 
  }
````

Can you elaborate more on this?
Also I would recommend defining the list on your strings in a `init` function.

Comment (kasimfareeda) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:34]: I was just putting in random words to test if it was working

Comment (setareh94) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:34]: Sounds good. Make sure to modify it for the final version.

**Comment by kasimfareeda** [comment]
> lgtm.
> 
> 1. Make sure to resolve the conflicts
> 2. Make sure that GSON is working fine I didn't see you importing it in XML file.

I deleted the gson because i didn’t use it so there was no need

**Pull request merged**